src/webapp/components/data-submission/Completed.tsx

```diff
@@ -0,0 +1,60 @@
+import React from "react";
+import styled from "styled-components";
+import { glassColors } from "../../pages/app/themes/dhis2.theme";
+import i18n from "@eyeseetea/d2-ui-components/locales";
+import { CompleteButtons } from "./CompleteButtons";
+
+export const Completed: React.FC = () => {
+    return (
+        <ContentWrapper>
+            <p className="intro">{i18n.t("Thank! your data for now is uploaded in our system")}</p>
+            <div className="call-name">
+                <span>2020</span>
+                <span>Spain</span>
+            </div>
+            <Section className="summary">
+                <p>Any other userful information here?</p>
+            </Section>
+            <Section className="summary">
+                <p>Info abou the previously updated quarter and the missing quarters</p>
```

Comment:
Missing i18n, but I guess there are temporal literals

Reply:
to be followed-up by this task: https://app.clickup.com/t/860pp72xz